### ISSUE TITLE
LUA-Error when VJ_PlayerDeath gets called without an attacker

### ISSUE THREAD

**Issue by CubeMath** [DrVrej/VJ-Base#89]
There is a gamemode that has nothing to do with the VJ Base mod, but has it installed.
That gamemode calls a Hook named "PlayerDeath" where their attacker is **nil**.
The function VJ_NPCPLY_DEATH causes the LUA-Error.

Can you check if the attacker is nil?

Something like this:
```
local function VJ_NPCPLY_DEATH(npc, attacker, inflictor)
	if attacker != nil && attacker.IsVJBaseSNPC == true then
		attacker:DoKilledEnemy(npc, attacker, inflictor)
		attacker:DoEntityRelationshipCheck()
	end
end
```

**Comment by DrVrej** [comment]
Thanks for the report, will be fixed soon.

**Comment by nsherbina1999** [comment]
If I spawn friendly and hostile NPCs with VJ_AK47, if someone kills each other, game hangs. Is this it?

**Comment by DrVrej** [comment]
Fixed in dba5875435028226152ebb2ddd340075a8d03932, thanks again for the report!

@nsherbina1999 No that sounds like a conflicting addon.
